railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -32,6 +32,14 @@ RUN curl -sL https://github.com/nodenv/node-build/archive/master.tar.gz | tar xz
     rm -rf /tmp/node-build-master
 
 <% end -%>
+
+<% if using_bun? -%>
+ENV BUN_INSTALL=/usr/local/bun
+ENV PATH=/usr/local/bun/bin:$PATH
+RUN curl -fsSL https://bun.sh/install | bash -s
```

Comment:
Can we freeze this to a version? Doesn't need to be a patch level, but I don't want to generate something using bun 2.0 if Rails isn't compatbile with it.

Reply:
Yes

Reply:
@rafaelfranca see https://github.com/rails/rails/pull/49241/commits/a2fb1b8fa0aa1e94f64049fcf390af9f17efb2ca

There isn't a way to truly lock the version, but this at least ensures the Dockerfile generated at install time is going to be locked to the version of Bun they had on their system which they can modify later.